packages/svelte2tsx/index.d.ts

```diff
@@ -195,3 +195,5 @@ export namespace InternalHelpers {
         paramsPath: string;
     }
 }
+
+export const RENDER_NAME:string;
```

Comment:
Can we expose this from a sub module, or via the helpers object that is not subject to semver? Wanna make sure that the "public public" API stays clean.

Reply:
Oh yeah better...just did it in a new commit